src/api/collections/articles.ts

```diff
@@ -46,6 +46,26 @@ interface ArticlesParams {
 
 const client = HttpClient.getInstance()
 
+function getArticles({
+  tag,
+  limit,
+  pageOffset,
+  options,
+}: {
+  tag: string
+  limit: number
+  pageOffset?: number
+  options?: RequestOptions
+}) {
+  const request = client.callPoliNetwork<Articles>({
+    url: "/v1/articles",
+    method: "GET",
+    params: { tag, limit, sort: "date", platform: 1, pageOffset: pageOffset },
+    ...options,
+  })
+  return mapAxiosRequest(request, res => res.articles[0])
+}
+
 /**
```

Comment:
la chiamata `getFromOffsetByTag` deve restituire una lista di articoli, diversamente da `getLastArticleByTag` che restituisce un singolo articolo.

Attualmente entrambe le chiamate restituiscono un singolo articolo generando errori.

Ciò che ti consiglio di fare è restituire semplicemente `request` dalla funzione `getArticles` e fare il mapping in maniera indipendente dentro le singole funzioni `getFromOffsetByTag` e `getLastArticleByTag`.

Fammi sapere se hai dubbi.

Reply:
progressi in questo fronte? @angeousta
perché altrimenti se non funziona chiudo la PR